api/kessel/inventory/v1beta2/reporter_data.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package kessel.inventory.v1beta2;
+
+option go_package = "github.com/project-kessel/inventory-api/api/kessel/inventory/v1beta2";
+option java_multiple_files = true;
+option java_package = "org.project_kessel.api.inventory.v1beta2";
+
+message ReporterData{
+  string reporter_type = 1;
```

Comment:
Should we take this opportunity to rename Reporter Type to Reporter ID?

Today we have...
```
Reporter Type = OCM, ACM, etc.
Reporter ID = "Client data" i.e. Postman, cURL, etc.
```

This seems misleading as an ID should be a unique identifier, like the Service Providers name. I think like we should do the following...
```
Reporter ID = OCM, ACM, etc.
Reporter Type or Reporter Client = "Client Data" i.e. Postman, cURL, etc.
```

Reply:
I have been meaning to get some clarification about this from Alec and Jaime anyway, as to what each of these fields mean and that can inform our naming. I will do that first and see if it aligns with your suggestions.